app/enterprise/2.1.x/deployment/installation/kong-for-kubernetes.md

```diff
@@ -1,20 +1,30 @@
 ---
 title: Installing Kong for Kubernetes Enterprise
 skip_read_time: true
+redirect_from: "/enteprise/2.1.x/kong-for-kubernetes/install"
 ---
 
 ## Introduction
-Kong for Kubernetes Enterprise provides most Kong Enterprise plugins and runs without a database, but does not include other Kong Enterprise features (Kong Manager, Dev Portal, Vitals, etc).
+Kong for Kubernetes Enterprise provides most Kong Enterprise plugins and runs
+without a database, but does not include other Kong Enterprise features (Kong
+Manager, Dev Portal, Vitals, etc).
+
+This installation topic walks you through a DB-less deployment on
+Kubernetes. For the full range of Enterprise features, see the database-backed
```

Comment:
It seems a bit odd to abbreviate for DB-less and not for database-backed. Maybe these should be with database and without database? That would also avoid having to use hyphens.

Reply:
I'm not sure how these conventions came around, but "DB-less mode" or simply "DB-less" is how we always refer to Kong running without a database; see https://docs.konghq.com/2.2.x/db-less-and-declarative-config/.

This phrasing is also mirroring the K8s deployment options breakdown: https://docs.konghq.com/enterprise/2.1.x/kong-for-kubernetes/deployment-options/

Will rephrase the second sentence instead to say:

"For the full range of Enterprise features, you need a database-backed or a hybrid deployment. See the instructions for [link], which walk you through a deployment with a database."